src/state/ledger/mod.rs

```diff
@@ -311,7 +311,7 @@ mod tests {
         let mut account = Account::empty(public_key.clone());
         account.balance = Amount(10); // Set the balance explicitly
         let mut accounts = HashMap::new();
-        accounts.insert(public_key.clone(), account);
+        accounts.insert(public_key.clone(), account.clone());
```

Comment:
Above line 313, write `let account_before = account.clone()`. Then you can eliminate this clone on 314.

Reply:
Did this